Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -40,6 +47,19 @@ def gistify_logs(f)
     puts url if url
   end
 
+  def brief_build_info(f)
+    build_time_str = f.logs.ctime.strftime("%Y-%m-%d %H:%M:%S")
+    hostname = Socket.gethostname
```

Comment:
Maybe put this line inside if branch below. Also, I wonder do we really need an extra option?